librz/core/cmd.c

```diff
@@ -4760,55 +4761,123 @@ DEFINE_HANDLE_TS_FCN_AND_SYMBOL(redirect_command) {
 	return res;
 }
 
+typedef struct _search_help {
+	bool color;
+	RzStrBuf *sb;
+	PJ *pj;
+} RzHelpSearch;
+
+static bool help_search_cmd_desc_entry(RzCmd *cmd, const RzCmdDesc *cd, void *user) {
+	rz_return_val_if_fail(cd, false);
+	if (cd->type != RZ_CMD_DESC_TYPE_ARGV &&
+		cd->type != RZ_CMD_DESC_TYPE_ARGV_MODES) {
+		return true;
+	}
+	RzHelpSearch *hs = (RzHelpSearch *)user;
+	if (hs->pj) {
+		rz_cmd_get_help_json(cmd, cd, hs->pj);
+	} else {
+		rz_cmd_get_help_strbuf(cmd, cd, hs->color, hs->sb);
+	}
+	return true;
+}
+
+RZ_IPI RzCmdStatus rz_cmd_help_search_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	bool is_interactive = rz_cons_is_interactive();
+	rz_cons_set_interactive(false);
```

Comment:
why this? is it to avoid the "Do you want to print X lines?"? I think it still makes sense to have that warning.

Reply:
Because before i was calling `<cmd>?` and some like `V` didn't like it and started the visual mode

Reply:
Is it still necessary?